Explorer/Assets/DCL/Editor/EntityBrowserWindow.cs

```diff
@@ -0,0 +1,393 @@
+using Arch.Core;
+using Arch.Core.Utils;
+using Global.Dynamic;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Pool;
+using Object = UnityEngine.Object;
+
+namespace DCL.Editor
+{
+    public sealed class EntityBrowserWindow : EditorWindow
```

Comment:
The left part should be flexible, otherwise names are truncated and it's impossible to understand what's behind them
![image](https://github.com/user-attachments/assets/0a529a93-e16c-4dbf-99cf-17f57d2eabd7)